### ISSUE TITLE
There is a pixel from original skin peeking under 2x2 eye model

### ISSUE THREAD

**Issue by Ferninglin** [IthanMendoza/Fresh-Moves#221]
![2017_06_12_skin_2017061208013776175V2fff](https://github.com/user-attachments/assets/5fa13bef-7e67-464c-a2a9-a3eed3e554de)
![2024-12-16_04 51 23](https://github.com/user-attachments/assets/9abf1cc8-989c-4d4f-aca4-416af94a7003)

I followed the instructions to my eye model + eyelashes, and I can see it worked in the game, but somehow the model is a pixel "higher" than the original skin's eyes. I saw some players had a similar problem, but couldn't find the solution myself. Please help!

**Comment by STRYTUS** [comment]
Seconding this tested both in 1.20.1 and 1.21.3, also including from other issues opened talking about the outer layer problem.
Tested with no other texture packs active but also tried [fresh animations](https://modrinth.com/resourcepack/fresh-animations) and both works fine together. though the eye pixel and outer layer problems still do exists.

### TL;DR 
- [3d skin layers](https://modrinth.com/mod/3dskinlayers) and [first person model mod](https://modrinth.com/mod/first-person-model/versions) works fine and doesn't conflict
- both 1.20.1 and 1.21.3 has the one pixel showing below the **eye problem**
- only 1.21.3 has the outer layer bug, 1.20.1 is safe from it
- both the eyeless and expressive version has the **outer layer problems** on 1.21.3


### Modlists
**(1.20.1)**
- entity_model_features_fabric_1.20.1-2.2.6
- entity_texture_features_fabric_1.20.1-6.2.9
- essential_1-3-5-5_fabric_1-20-1
- fabric-api-0.92.2+1.20.1
- firstperson-fabric-2.4.6-mc1.20.1
- iris-1.7.5+mc1.20.1
- lithium-fabric-mc1.20.1-0.11.2
- modmenu-7.2.2
- skinlayers3d-fabric-1.7.4-mc1.20.1
- sodium-fabric-0.5.11+mc1.20.1

1.21.3 has the same mods but just updated and are the latest available on that version currently, also removed 3d skin layers for  testing it out in 1.21.3

### Screenshots
**My edited skin**
![image_2024-12-17_030144423](https://github.com/user-attachments/assets/09fad988-3484-41a0-b113-da439bf4f374)


**1.20.1 (has 3d skin layers mod)**
without the resource pack and how it should look like
![Screenshot (756)](https://github.com/user-attachments/assets/d8f5979c-1fb7-481e-9b69-fd99b0712b0b)
one pixel below eye problem
![Screenshot (757)](https://github.com/user-attachments/assets/d62991e8-2afa-4f5a-87bb-8130da2d150d)


**1.21.3 (with no 3d skin layers mod)**
outer layer problem (eyeless version)
![Screenshot (759)](https://github.com/user-attachments/assets/fee5b5cf-38a9-4cd0-be7f-3973fdd31a71)
outer layer problem + one pixel below eye problem (expressive version)
![Screenshot (758)](https://github.com/user-attachments/assets/3292ef1d-00ff-4271-91e1-402585d76808)

**Comment by STRYTUS** [comment]
Found a fork from erinsc in the comment that fixed this but the eye problem still exists. Tested on 1.21.3 which has the outer layer problem.
[Link to fork by erinsc](https://github.com/erinsc/Fresh-Moves)
![image](https://github.com/user-attachments/assets/86010e13-3775-4758-80c4-3d5e3eda4b90)

### Videos showing movements
Original 
https://github.com/user-attachments/assets/9f816a51-96db-404e-a968-901a88539328

Forked fix
https://github.com/user-attachments/assets/ced4d0df-aeac-4b75-90a3-1e51ffd58d9b

**Comment by Ferninglin** [comment]
Hey, I think I found a solution for the eye-problem! If you have eyes on the lower mid, you should use the "bottom eyes" model. Somehow this model also goes a pixel higher than needed and just on the level the lower-mid eyes require. 
![Снимок экрана (718)](https://github.com/user-attachments/assets/f7b20ee7-6f83-4d50-8f75-e29c2b60f304)

Still not sure what to do with your outer layer problem or the lashes, but still... At least eyes are okay now. I will update if I figure something out
